### DISCUSSION TITLE
qca ssdk 12编译错误

### DISCUSSION THREAD

**Discussion by Lstions** [Boos4721/openwrt#163] [Bugs/问题反馈]
make的时候报错，qca ssdk  **“br_fdb_delete_by_netdev“** 函数 undefined

后来查了查qsdk内核代码，发现大佬没有添加相关函数。
相关的commit在此[linux: Delete fdb entry using netdev](https://git.codelinaro.org/clo/qsdk/oss/kernel/linux-ipq-5.4/-/commit/30d033d059313765500ef2061ba0a3200e3aea1a)

**Comment by Boos4721** [comment]
PR IT

**Comment by YIDAWN** [comment]
大佬，我的AX3600编译也报qca ssdk问题。怎么解决

**Reply by Lstions** [reply]
临时做了一个patch
https://github.com/Lstions/openwrt-boos/commit/82130c8721dad6b0fc2e55dce80135a760e683bc

**Reply by YIDAWN** [reply]
> 临时做了一个patch [Lstions@82130c8](https://github.com/Lstions/openwrt-boos/commit/82130c8721dad6b0fc2e55dce80135a760e683bc)

 make[3] -C package/lean/ipv6-helper compile
 make[3] -C feeds/luci/applications/luci-app-autoreboot compile
 make[2] package/install
**make -r world: build failed. Please re-run make with -j1 V=s or V=sc for a higher verbosity level to see what's going on
make: *** [/workdir/openwrt/include/toplevel.mk:230: world] Error 1**
 make[1] world
 make[2] target/compile
 make[3] -C target/linux compile
 make[2] buildinfo
 make[3] diffconfig buildversion feedsversion
 make[4] scripts/config/conf
 make[2] package/cleanup
 make[2] package/compile
 make[3] -C package/libs/libjson-c host-compile
 make[3] -C package/libs/libubox host-compile

加了你的这个文件，不报qca 的错误了，但是还是编译没成功。 这个能看出来是什么原因不

**Reply by Lstions** [reply]
我ax3600能编译成功

**Reply by gtcar2020** [reply]
用了补丁也不行

**Comment by kiddin9** [comment]
用这个patch编译成功了, 但是第一次启动后,再重启就会无限重启.

**Reply by Lstions** [reply]
有日志吗

**Reply by kiddin9** [reply]
确定是kernel的问题, 用5.15.49的kernel就没问题.

**Reply by MonwF** [reply]
> 确定是kernel的问题, 用5.15.49的kernel就没问题.

5.15.49编译了一版 带nss wifi速度只有400m。 你那边速度ok吗， 贴得很近

**Reply by 332505** [reply]
用5.15.56的内核遇到了同样的问题

**Reply by gtcar2020** [reply]
不要用这个源了，编译后反复重启，退回到5.15.49概览显示不正常，多拨也不能叠加网速。

**Reply by xiaiohuan** [reply]
> 不要用这个来源了，编译后回复重启，返回到5.15.49概览显示不正常，多拨也不能增加网速。

测试5.15.45内核编译没有问题且挺稳的 设备为红米AX6
